src/components/DateRangeInput/DoubleTextInput.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { ChangeEventHandler } from 'react';
```

Comment:
A grouped input component. Labels, names, placeholders and ids may get generated.